Cargo.toml

```diff
@@ -69,8 +74,10 @@ rust.unused_macro_rules = "warn"
 rust.unused_qualifications = "warn"
 rust.variant_size_differences = "warn"
 
-clippy.allow_attributes = "warn"
-clippy.allow_attributes_without_reason = "warn"
+# FIXME(color): I'm lazy about fixing this yet.
+clippy.allow_attributes = "allow"
```

Comment:
This should be a crate level exception, I think.

Reply:
I'd like to see the MSRV bump to 1.82 happen independently in preparation of this landing and so this sort of change won't be needed here.

Reply:
Between #65 and here, I've resolved the MSRV stuff.